reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -319,15 +319,12 @@ class StaticHtmlReporter : Reporter {
         val rowId = "issue-$rowIndex"
 
         val issues = (row.analyzerIssues + row.scanIssues).flatMap { it.value }
-
-        val worstSeverity = issues.map { it.severity }.min() ?: Severity.ERROR
-
         val areAllResolved = issues.isNotEmpty() && issues.all { it.isResolved }
 
         val cssClass = if (areAllResolved) {
             "ort-resolved"
         } else {
-            when (worstSeverity) {
+            when (issues.map { it.severity }.min() ?: Severity.ERROR) {
```

Comment:
Don't you think this was easier to read before inlining the variable, because the variable name helps to understand what this is about?

Reply:
Agree, let me get it back. I was kind of blindly following the IDE hint.